### PR TITLE
Update nightly and release icons from Figma

### PR DESCRIPTION
**Pull request by pierremtb**

### CODE REVIEW

File: assets/icon-nightly.ico
Comment (pierremtb) [inline assets/icon-nightly.ico:1]: Spent an absurd amount of time on this: I ended up converting those .ico files from the figma .png directly with https://convertico.com, as running ffmpeg as suggested [here](https://superuser.com/a/679559) (after adjusting the size down to 256x256 with Preview) led to weird ugly results once opened on windows, unsure why. Note that removing the .ico files altogether and using their auto conversion as mentioned at https://www.electron.build/icons.html#windows-nsis also leads to bad results.

**Comment by pierremtb** [comment]
Can wait for Monday @franknoirot, all good

File: assets/icon-nightly.ico
Comment (franknoirot) [reply assets/icon-nightly.ico:1]: That's so annoying!

**Review by franknoirot — APPROVED** [review]

File: assets/icon-nightly.ico
Comment (pierremtb) [reply assets/icon-nightly.ico:1]: Yeah I'm not fighting this today but wanted to make sure this was recorded somewhere haha. Thanks for approving!

**Pull request merged**